includes/admin.inc

```diff
@@ -529,6 +529,25 @@ function islandora_solr_admin_settings($form, &$form_state) {
     '#description' => t('Use labels instead of raw field names when displaying an advanced search in the Current Query block.'),
   );
 
+  $form['islandora_solr_tabs']['advanced_search_block']['islandora_solr_advanced_search_block_lucene_syntax_escape'] = array(
+    '#title' => t('Escape Lucene special characters'),
+    '#description' => t('Allow the use of lucene syntax string escaping on search terms'),
+    '#type' => 'checkbox',
+    '#default_value' => variable_get('islandora_solr_advanced_search_block_lucene_syntax_escape', TRUE),
+  );
+
+  $form['islandora_solr_tabs']['advanced_search_block']['islandora_solr_advanced_search_block_lucene_regex_default'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Default regular expression evaluated on search term'),
+    '#default_value' => variable_get('islandora_solr_advanced_search_block_lucene_regex_default', '/(\+|-|&&|\|\||!|\(|\)|\{|\}|\[|\]|\^|~|\*|\?|\:|\")/'),
```

Comment:
Would love to see this  being a CONSTANT so other solution packs (like facet pages) could be more consistent if possible